DEVELOPERS.md

```diff
@@ -1,42 +1,16 @@
 # MetaMask Compatibility Guide
```

Comment:
Should merge into `develop`.

Reply:
I didn't see a `develop` branch for this FAQ repository, only `master`. Maybe I need different permissions?

Reply:
No, this repo does not have a develop branch.